sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -269,7 +269,7 @@ org.apache.spark.sql.catalyst.parser.ParseException
   "errorClass" : "PARSE_SYNTAX_ERROR",
   "sqlState" : "42601",
   "messageParameters" : {
-    "error" : "'.'",
+    "error" : "end of input",
```

Comment:
Could you explain why this PR changes this error parameter?

Reply:
I will check this one